### PR TITLE
Profile Screen

### PR DESCRIPTION
**Pull request by Josetf2396**
image

### CODE REVIEW

**Comment by Josetf2396** [comment]
image

**Comment by lasharela** [comment]
We need button here as well. Action button

File: mobile/src/screens/profile/Profile.view.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+export const ProfileView: FC<TProfileViewProps> = ({
+  name,
+  email,
+  phoneNumber,
+  profileURL,
+  goBack,
+  saveButton,
+}) => {
+  return (
+    <>
```
Comment (lasharela) [inline mobile/src/screens/profile/Profile.view.tsx:17]: you don't need this

File: mobile/src/screens/profile/Profile.view.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+              placeholder={phoneNumber}
+              value={''}
+              onChangeText={function (value: string): void {
+                throw new Error('Function not implemented.');
+              }}
+            />
+          </Section>
+        </View>
+
+        <Button
```
Comment (lasharela) [inline mobile/src/screens/profile/Profile.view.tsx:72]: footer prop of Container component

File: mobile/src/screens/profile/profileDummyData.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+export const profileDummyData = [
```
Comment (lasharela) [inline mobile/src/screens/profile/profileDummyData.ts:1]: move to backend

File: mobile/src/screens/profile/Profile.view.tsx [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+              }}
+            />
+          </Section>
+          <Section>
+            <Text style={styles.text}>E-mail</Text>
+            <TextInput
+              type={'regular'}
+              placeholder={email}
+              value={''}
+              onChangeText={function (value: string): void {
```
Comment (lasharela) [inline mobile/src/screens/profile/Profile.view.tsx:54]: function - keep consistency and use ()=>

onChangeText={(value: string): void => {

File: mobile/src/screens/profile/Profile.view.tsx [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+                throw new Error('Function not implemented.');
+              }}
+            />
+          </Section>
+          <Section>
+            <Text style={styles.text}>Phone number</Text>
+            <TextInput
+              type={'regular'}
+              placeholder={phoneNumber}
+              value={''}
```
Comment (lasharela) [inline mobile/src/screens/profile/Profile.view.tsx:64]: no empty string please

**Review by lasharela — CHANGES_REQUESTED** [review]

**Pull request closed without merging**